server/routes/auth/index.ts

```diff
@@ -68,6 +70,27 @@ router.get("/redirect", auth(), async (ctx) => {
   );
 });
 
+router.get("/transfer", auth(), async (ctx) => {
```

Comment:
This method scares me, lets chat on alternatives. We can probably use the `transfer` JWT like we do from hopping from the apex -> subdomain.

https://github.com/outline/outline/blob/863f22750ffb86fec628901a8380d285e5c12ab3/server/utils/jwt.ts#L50

Reply:
Interesting, from reading the code, this seemed to be the most straightforward way to initiate the transfer token stuff. Maybe we could do a one time pad on success of the team creation step?

Reply:
Coming back to this… yep, it's still scary 😆. We should return the redirect url from the team creation api call which includes the temporary authentication token and uses the path that already exists… all you need is the new `team.url` and `user` on the server to get it:

`${team.url}/auth/redirect?token=${user.getTransferToken()}`

We can then remove this endpoint entirely If I'm understanding everything correctly

Reply:
Yep, I think you're right